homeassistant/components/google_assistant/trait.py

```diff
@@ -675,3 +676,77 @@ def can_execute(self, command, params):
         await self.hass.services.async_call(lock.DOMAIN, service, {
             ATTR_ENTITY_ID: self.state.entity_id
         }, blocking=True)
+
+
+@register_trait
+class FanSpeedTrait(_Trait):
+    """Trait to control speed of Fan.
+
+    https://developers.google.com/actions/smarthome/traits/fanspeed
+    """
+
+    name = TRAIT_FANSPEED
+    commands = [
+        COMMAND_FANSPEED
+    ]
+
+    speed_synonyms = {
+        "off": ['stop', 'off'],
+        "low": ['slow', 'low', 'slowest', 'lowest'],
+        "medium": ['medium', 'mid', 'middle'],
+        "high": ['high', 'max', 'fast', 'highest', 'fastest', 'maximum']
+    }
+
+    @staticmethod
+    def supported(domain, features):
+        """Test if state is supported."""
+        if domain != fan.DOMAIN:
+            return False
+
+        return features & fan.SUPPORT_SET_SPEED
+
+    def sync_attributes(self):
+        """Return speed point and modes attributes for a sync request."""
+        modes = self.state.attributes.get(fan.ATTR_SPEED_LIST, [])
+        speeds = []
+        for mode in modes:
+            speed = {
+                "speed_name": mode,
+                "speed_values": [{
+                    "speed_synonym": self.speed_synonyms.get(mode),
+                    "lang": 'en'
+                }]
+            }
+            speeds.append(speed)
+
+        return {
+            'availableFanSpeeds': {
+                'speeds': speeds,
+                'ordered': True
+            },
+            "reversible": True if (
```

Comment:
Instead of `True if`, use `bool(…)` or define the if-statement above the `return`. Inline if-statements are not very readable.

Reply:
Chose to use bool(). Didn't do that previously due to an earlier mistake I had already fixed.  
Fixed in 60e677a921f7f5ca4b38e86caddf2a0590d9be84